pkg/exchange/bybit/exchange.go

```diff
@@ -169,3 +175,82 @@ func (e *Exchange) QueryOpenOrders(ctx context.Context, symbol string) (orders [
 
 	return orders, nil
 }
+
+func (e *Exchange) SubmitOrder(ctx context.Context, order types.SubmitOrder) (*types.Order, error) {
+	if len(order.Market.Symbol) == 0 {
+		log.Warnf("empty symbol, orderClientId: %s", order.ClientOrderID)
+		return nil, fmt.Errorf("empty symbol")
```

Comment:
maybe just:

```
fmt.Errorf("order.Market is required: %+v", order)
```

and remove the warning, because SubmitOrder is a critical operation, we should stop the when error happens